### PR TITLE
Repo uniqueness ls

### PR DESCRIPTION
**Pull request by lrs8810**
#16 
* Update specs: Add edge case for repos
* Update view: Add CCS class
* Update model: Add spacing 
* All test passing

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -3,6 +3,7 @@ class User < ApplicationRecord
   has_many :videos, through: :user_videos
 
   validates :email, uniqueness: true, presence: true
+
```
Comment (davidlopez) [inline app/models/user.rb:6]: Add spacing ✅

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -10,7 +10,7 @@
   <% if !user_facade.user.connected? %>
     <%= link_to 'Connect to Github', github_info_path, method: 'get', class: "btn btn-primary mb1 bg-teal" %>
   <% else %>
-    <section>
+    <section class='github-info'>
```
Comment (davidlopez) [inline app/views/users/show.html.erb:13]: Add class ✅

File: spec/features/user/oauth/user_can_see_github_info_spec.rb [hunk 3]
```diff
@@ -23,15 +23,15 @@
     @following_3 = Following.new({login:'Anna3' , html_url: "https://github.com/lrs8810/brownfield-of-dreams"})
 
     @followings = [@following_1, @following_2, @following_3]
+  end
 
+  it 'I can see 5 repos that belong to me' do
```
Comment (davidlopez) [inline spec/features/user/oauth/user_can_see_github_info_spec.rb:28]: Add edge case ✅

**Review by davidlopez — APPROVED** [review]
Greate work with the testing
